tests/Aspire.Npgsql.EntityFrameworkCore.PostgreSQL.Tests/ConfigurationTests.cs

```diff
@@ -30,16 +25,6 @@ public void TracingIsEnabledByDefault()
     public void MetricsAreEnabledByDefault()
         => Assert.True(new NpgsqlEntityFrameworkCorePostgreSQLSettings().Metrics);
 
-    [Fact]
-    public void MaxRetryCountIsSameAsInTheDefaultNpgsqlPolicy()
-    {
-        DbContextOptionsBuilder<TestDbContext> dbContextOptionsBuilder = new();
-        dbContextOptionsBuilder.UseNpgsql("fakeConnectionString");
-        TestDbContext dbContext = new(dbContextOptionsBuilder.Options);
-
-        Assert.Equal(new WorkaroundToReadProtectedField(dbContext).RetryCount, new NpgsqlEntityFrameworkCorePostgreSQLSettings().MaxRetryCount);
-    }
-
     public class WorkaroundToReadProtectedField : NpgsqlRetryingExecutionStrategy
```

Comment:
This can be replaced by the common one now. Or just removed since it isn't used anymore.